File: app/src/main/java/com/ragdroid/dahaka/activity/home/profile/ProfileContract.java
package com.ragdroid.dahaka.activity.login;

import com.ragdroid.dahaka.mvp.BasePresenter;
import com.ragdroid.dahaka.mvp.BaseView;

/**
 * Created by garimajain on 13/08/17.
 */

public interface LoginContract {

    interface View extends BaseView {

    }

    interface Presenter extends BasePresenter {
        void onSubmitClicked(LoginModel loginModel);
    }

}

File: app/src/main/java/com/ragdroid/dahaka/activity/home/profile/ProfileContract.kt
package com.ragdroid.dahaka.activity.home.profile

import com.ragdroid.dahaka.mvp.BasePresenter
import com.ragdroid.dahaka.mvp.BaseView

/**
 * Created by garimajain on 13/08/17.
 */

interface ProfileContract {

    interface View : BaseView {

        fun showModel(model: ProfileModel)
    }

    interface Presenter : BasePresenter<View>

}
